cmd/kubelet/app/server.go

```diff
@@ -740,7 +745,7 @@ func run(ctx context.Context, s *options.KubeletServer, kubeDeps *kubelet.Depend
 	cgroupRoots = append(cgroupRoots, nodeAllocatableRoot)
 	kubeletCgroup, err := cm.GetKubeletContainer(s.KubeletCgroups)
 	if err != nil {
-		klog.InfoS("Failed to get the kubelet's cgroup. Kubelet system container metrics may be missing.", "err", err)
+		logger.Info("Failed to get the kubelet's cgroup. Kubelet system container metrics may be missing.", "err", err)
```

Comment:
```suggestion
		logger.Info("Failed to get the kubelet's cgroup. Kubelet system container metrics may be missing", "err", err)
```

Reply:
Thank you for your review.
I fixed here https://github.com/kubernetes/kubernetes/pull/127887/commits/2ac71d903967e1f1a3eab6b473a9c9cc0a0f02a8 👍